layers/transportation_name/transportation_name.sql

```diff
@@ -28,9 +28,9 @@ CREATE OR REPLACE FUNCTION layer_transportation_name(bbox geometry, zoom_level i
 AS
 $$
 SELECT geometry,
-       name,
-       COALESCE(name_en, name) AS name_en,
-       COALESCE(name_de, name, name_en) AS name_de,
+       tags->'name',
```

Comment:
Just `SELECT tags->'name'` return unnamed column. Maybe it will be better to explicit the name.
```suggestion
       tags->'name' AS name,
```